### DISCUSSION TITLE
Parsing CRNN ONNX model

### DISCUSSION THREAD

**Discussion by raushani2v** [open-mmlab/mmocr#2016] [Q&A]
Hi,

I am trying to parse the output of the CRNN onnx model. The output that I get from the onnx model is a list of length 1. The shape of the element of the list is (1, 26, 11); where 1 is batch size, 26 is sequence length (i.e, maximum number of characters in a sequence), and I guess 11 is the num_classes+1 because I have used a dict of length 10. i.e, dict='0123456789'

My Question is what does this 11th value(logits) represent ? Is it the probability of EOS or a separator or something else?

I tried to parse using following code:

```
self.sequence_length = 26
self.dictionary = "0123456789"

output_tensor = self.session.run(output_names, {input_names[0]: blob.astype(np.float32)})
print(output_tensor[0].shape)  # (1, 26, 11)

characters = len(self.dictionary) + 1  # separator

# CRNN output parsing..
max_indices = []
for outer in range (0, self.sequence_length):
            character_index = -1
            character_value = 0
            for inner in range(0, characters):
                value = output_tensor[0][0][outer][inner]

                if (value > character_value):
                    character_value = value
                    character_index = inner
            max_indices.append(character_index)

print(max_indices)
```

max_indices returns [10, 10, 10, 9, 9, 10, 10, 10, 9, 9, 10, 10, 10, 10, 8, 10, 10, 10, 10, 4, 4, 10, 10, 10, 10, 10] for ground truth '9984'

What does this index 10 mean?

How can I get the result as '9984' from the above max_indices list?
